README.md

```diff
@@ -191,6 +191,20 @@ The shared storage worklet invocation methods (`addModule`, `run`, and `selectUR
 *  `sharedStorage.context`
     *   From inside a worklet created inside a [fenced frame](https://github.com/wicg/fenced-frame/), returns a string of contextual information, if any, that the embedder had written to the [fenced frame](https://github.com/wicg/fenced-frame/)'s [FencedFrameConfig](https://github.com/WICG/fenced-frame/blob/master/explainer/fenced_frame_config.md) before the [fenced frame](https://github.com/wicg/fenced-frame/)'s navigation.
     *   If no contextual information string had been written for the given frame, returns undefined.
+*   `interestGroups()`
```

Comment:
This function is on the `sharedStorage` object?

Reply:
It's in the global context of the shared storage worklet